adam-core/src/main/scala/org/bdgenomics/adam/rdd/read/FlagStat.scala

```diff
@@ -93,15 +94,15 @@ object FlagStat {
         val (primaryDuplicates, secondaryDuplicates) = DuplicateMetrics(p)
         new FlagStatMetrics(1,
           primaryDuplicates, secondaryDuplicates,
-          b2i(p.getReadMapped),
-          b2i(p.getReadPaired),
-          b2i(p.getReadPaired && p.getFirstOfPair),
-          b2i(p.getReadPaired && p.getSecondOfPair),
-          b2i(p.getReadPaired && p.getProperPair),
-          b2i(p.getReadPaired && p.getReadMapped && p.getMateMapped),
-          b2i(p.getReadPaired && p.getReadMapped && !p.getMateMapped),
-          b2i(mateMappedToDiffChromosome),
-          b2i(mateMappedToDiffChromosome && p.getMapq >= 5),
+          b2i(b(p.getReadMapped)),
```

Comment:
These changes look reasonable but I think we should pull these changes out to another PR.